import-tags.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+public=''
+
+while getopts c:t:pu: flag
```

Comment:
getopts are complicated, shouldn't this be `c:t:p:u`? Could we also add long options here that correspond to the snoop ones?